packages/contracts-bedrock/src/L2/ETHLiquidity.sol

```diff
@@ -24,6 +27,9 @@ contract ETHLiquidity is ISemver {
     /// @notice Emitted when an address mints ETH liquidity.
     event LiquidityMinted(address indexed caller, uint256 value);
 
+    /// @notice Event to emit when funds are received
+    event LiquidityFunded(address indexed funder, uint256 amount);
+
     /// @notice Semantic version.
     /// @custom:semver 1.0.1
     string public constant version = "1.0.1";
```

Comment:
can you bump the minor semver here? `1.1.0`

Reply:
Done